packages/ui/src/common/components/icons/locks/LockIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { Icon } from '../Icon'
+
+interface Props {
+  className?: any
+}
+
+export const LockIcon = React.memo(({ className }: Props) => (
+  <Icon
+    preserveAspectRatio="xMidYMid meet"
+    color="currentColor"
+    className={className}
+    size="16"
+    viewBox="0 0 24 24"
```

Comment:
Could you check why this was needed? If I set `viewBox="0 0 16 16"` the lock was cropped.